target/linux/ath79/dts/qca9531_telco_electronics_tel-t1.dts

```diff
@@ -0,0 +1,147 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca953x.dtsi"
+
+/ {
+	compatible = "telco-electronics,t1", "qca,qca9531";
+	model = "Telco Electronics T1";
+	
+	aliases {
+		led-boot = &system_led;
+		led-failsafe = &system_led;
+		led-upgrade = &system_led;
+	};
+	
+	keys {
+		compatible = "gpio-keys";
+		poll-interval = <20>;
+
+		reset {
+			label = "reset";
+			linux,code = <KEY_RESTART>;
+			gpios = <&gpio 17 GPIO_ACTIVE_LOW>;
+		};
+	};
+	
+	leds {
+		compatible = "gpio-leds";
+		pinctrl-names = "default";
+		pinctrl-0 = <&jtag_disable_pins &led_wan_pin>;
+
+		wan {
+			label = "t1:blue:wan";
+			gpios = <&gpio 4 GPIO_ACTIVE_LOW>;
+		};
+
+		lan {
+			label = "t1:blue:lan";
+			gpios = <&gpio 16 GPIO_ACTIVE_LOW>;
+		};
+
+		wlan {
+			label = "t1:blue:wlan";
+			gpios = <&gpio 0 GPIO_ACTIVE_LOW>;
+			linux,default-trigger = "phy0tpt";
+		};
+
+		system_led: signal1 {
+			label = "t1:blue:signal1";
+			gpios = <&gpio 2 GPIO_ACTIVE_LOW>;
+		};
+
+		signal2 {
+			label = "t1:blue:signal2";
+			gpios = <&gpio 3 GPIO_ACTIVE_LOW>;
+		};
+
+		signal3 {
+			label = "t1:blue:signal3";
+			gpios = <&gpio 15 GPIO_ACTIVE_LOW>;
+		};
+	};
+};
+
+&uart {
+	status = "okay";
+};
+
+&usb0 {
+	#address-cells = <1>;
+	#size-cells = <0>;
+	status = "okay";
+
+	hub_port: port@1 {
```

Comment:
This node is totally useless if you don not have a led with a default-trigger to this set...
So drop the full subnode port@1 of usb0.